rts/Rendering/Fonts/CFontTexture.cpp

```diff
@@ -633,6 +699,166 @@ CFontTexture::~CFontTexture()
 #endif
 }
 
+/***
+ *
+ * Add a fallback font
+ *
+ * @param fontfile VFS path for the font
+ * @param clearMode ClearGlyphMode
+ */
+bool CFontTexture::AddFallbackFont(const std::string& fontfile, ClearGlyphMode clearMode)
```

Comment:
clearMode seems redundant CFontTexture::AddFallbackFont(), should really belong to `ClearFallbackFonts(ClearGlyphMode clearMode)`